.github/workflows/bump-n-release.yml

```
@@ -38,16 +38,20 @@ jobs:
           token: ${{ secrets.BUMP_N_RELEASE }}
           fetch-depth: 0
           fetch-tags: true
-      - uses: actions/setup-python@v5
-        with:
-          # here we need v3.10+
-          python-version: 3.x
+
       - run: rustup update --no-self-update
 
+      - name: Setup nushell
+        uses: hustcer/setup-nu@v3
+        with:
+          version: "*"
```

Pin this to a specific version of nushell. As of this writing, the latest is `0.106.0`:
```suggestion
          version: '0.106.0'
```